Core/composer.json

```diff
@@ -14,11 +14,10 @@
         "psr/http-message": "1.0.*"
```

Comment:
IMHO with PHP 7.4 here can make psr/http-message = "^1.0" (so allow install v1.1)

Reply:
Unfortunately this wouldn't have any kind of effect until `guzzlehttp/psr7` also supports it (see [here](https://github.com/guzzle/psr7/blob/master/composer.json#L55))

Reply:
At least "^1.0" - also good, thanks)